languages/lang-ja/Naming.ja.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<naming Lang="ja">
+<naming Lang="ja" Load="Merge">
 
   <namestyles>
 
-    <namestyle Name="Qudish (ja)" Format="AsIs">
-      <scopes>
+    <namestyle Name="Qudish" Format="AsIs">
```

Comment:
so we don't need the ` (ja)` since we want to reuse the scopes/attributes from the parent in base file